include/algorithms/boosting/boosting_training_batch.h

```diff
@@ -68,10 +68,10 @@ namespace interface1
  *      - \ref classifier::training::interface1::Input "classifier::training::Input" class
  *      - \ref classifier::training::interface1::Result "classifier::training::Result" class
  */
-class DAAL_EXPORT Batch : public classifier::training::Batch
+class DAAL_EXPORT Batch : public classifier::training::interface1::Batch
```

Comment:
Does it mean that this class is deprecated as there is no interface2::Batch?

Reply:
boosting namespace is fully deprecated